.coveragerc

```diff
@@ -1,2 +1,27 @@
 [run]
-source=app
+branch = True
+source = app
+omit =
+    _db/*
+    local_config.py
+    setup.py
+    migrations/*
+    tasks/*
+    virtualenv/*
```

Comment:
tests?

Reply:
It self-excludes tests from being tested since that is the testing folder.  We use a plugin to PyTest so it will also tell us if there are test files that are being skipped.